pyiron_base/project/delayed.py

```diff
@@ -0,0 +1,223 @@
+import hashlib
+import inspect
+import re
+
+import cloudpickle
+
+
+def get_function_parameter_dict(funct):
+    return {
+        k: None if v.default == inspect._empty else v.default
+        for k, v in inspect.signature(funct).parameters.items()
+    }
+
+
+def get_hash(binary):
+    # Remove specification of jupyter kernel from hash to be deterministic
+    binary_no_ipykernel = re.sub(b"(?<=/ipykernel_)(.*)(?=/)", b"", binary)
+    return str(hashlib.md5(binary_no_ipykernel).hexdigest())
+
+
+def get_graph(obj, obj_name=None, nodes_dict={}, edges_lst=[], link_node=None):
+    if isinstance(obj, DelayedObject):
+        try:
+            obj_name = (
+                obj._function.__name__ + "_" + get_hash(binary=cloudpickle.dumps(obj))
+            )
```

Comment:
```suggestion
            obj_name = obj._function.__name__
```

Since hash is added to `obj_name`, it's not necessary here.